podvm/files/etc/systemd/system/kata-agent.service

```diff
@@ -6,6 +6,8 @@ After=netns@podns.service process-user-data.service
 
 [Service]
 ExecStartPre=mkdir -p /run/kata-containers
+ExecStartPre=mkdir -p /run/peerpod
+ExecStartPre=-ln -s /etc/agent-config.toml /run/peerpod/agent-config.toml
```

Comment:
I think we don't need this anymore

Reply:
Yeah, I'll rebase against your latest changes